test/test-directives.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const Test = require('tape');
+
+const GraphQLComponent = require('../lib/index');
+const Directives = require('../lib/directives');
+
+Test('test component', (t) => {
```

Comment:
I think this test needs to validate `execute` against a component takes into account the directive, which will validate it ended up in the schema that gets created.